qml/pages/SettingsOverlay.qml

```diff
@@ -183,7 +183,7 @@ Item {
     DockedListView {
         id: panelFocus
         model: modelFocus
-        selectedItem: settings.mode.focus
+        selectedItem: page._manualModeSelected ? Camera.FocusManual : camera.focus.focusMode
```

Comment:
Is this change needed? You always have settings.mode.focus as the selected mode, in the new code you only set camera focus mode to Auto, not the settings mode.